contrib/!window-management/perspectives/packages.el

```diff
@@ -1,107 +1,163 @@
 (setq perspectives-packages
-  '(
-    perspective
-    persp-projectile
-    ))
+      '(perspective
+        persp-projectile))
 
 (defun perspectives/init-perspective ()
   (use-package perspective
+    :defer
     :commands (custom-persp
                persp-add-buffer
                persp-set-buffer
                persp-kill
                persp-remove-buffer
+               persp-mode
                ;; persp-cycle-next
                ;; persp-cycle-prev
                persp-rename
                persp-switch
-               projectile-persp-bridge
-               )
+               projectile-persp-bridge)
```

Comment:
`projectile-persp-bridge` is part of `persp-projectile`, should be under `perspectives/init-persp-projectile`